Client/mods/deathmatch/logic/CIFPEngine.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __CIFPENGINE_H
```

Comment:
File is missing the copyright/etc. comment at the top.
Also, use **#pragma once**